website/src/app/documentation/demos/daterangepicker/daterangepicker.demo.html

```diff
@@ -72,6 +73,26 @@ <h4><code class="clr-code">clr-daterangepicker-container</code> element</h4>
               List of presets. By default this list is empty, which hides the presets section in the UI.
             </td>
           </tr>
+          <tr>
+            <td class="left"><strong>[timeSelection]</strong></td>
+            <td class="left clr-hidden-xs-down"><code class="clr-code">Boolean</code></td>
+            <td class="clr-hidden-xs-down"><code class="clr-code"></code></td>
+            <td class="left">Set to true, if time selection should be activated.</td>
+          </tr>
+          <tr>
+            <td class="left"><strong>[activateSeconds]</strong></td>
+            <td class="left clr-hidden-xs-down"><code class="clr-code">Boolean</code></td>
+            <td class="clr-hidden-xs-down"><code class="clr-code"></code></td>
+            <td class="left">Set to true, if seconds should be shown on timeselection.</td>
+          </tr>
+          <tr>
+            <td class="left"><strong>[presetsTime]</strong></td>
```

Comment:
could we rename this to presetsDateTime, to make it clear that we could replace it with the presets, not only use it additive
and also give an advice if we should use presets and presetsTime in combination or only presetsTime if we have timeSelection set to true